painting/src/main/java/com/ataulm/artcollector/painting/ui/PaintingActivity.kt

```diff
@@ -50,7 +50,7 @@ class PaintingActivity : AppCompatActivity() {
             painting.loadImageIfDifferent()
         })
 
-        imageView.setOnViewTapListener { _, _, _ ->
+        imageView.setOnClickListener {
```

Comment:
I updated PhotoView to the latest version too - the only things that changed was this and the namespace for the view itself.